helpers/structure/create_structure.py

```diff
@@ -0,0 +1,16 @@
+from api.structure.models import Structure as StructureModel
+
+
+def create_structure(**kwargs):
+    """ Creates a single node in the office structure model"""
+    structure = StructureModel(
+        structure_id=kwargs['structure_id'],
```

Comment:
Kindly use `kwargs.get()` instead of `kwargs[]`

Reply:
@Bonifase Do you mean this for all the fields or just `structure_id` ?

Reply:
all the fields

Reply:
Okay, Done.